config-example/docker/myhost/.env

```
@@ -9,6 +9,7 @@ STORAGE_CALIBRE_LIBRARY=${STORAGE_LOCAL_MEDIA}/books
 STORAGE_CALIBRE_LIBRARY_HUN=${STORAGE_LOCAL_MEDIA}/books-hun
 STORAGE_WEBDAV=${STORAGE_LOCAL_MEDIA}/webdav
 STORAGE_KIWIX=${STORAGE_LOCAL_MEDIA}/kiwix
+STORAGE_OBSIDIAN=${STORAGE_LOCAL_MEDIA}/obsidian
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The STORAGE_OBSIDIAN key should go before the STORAGE_WEBDAV key

(UnorderedKey)